shared/src/main/scala/amf/client/remod/AMFResolver.scala

```diff
@@ -5,8 +5,8 @@ import amf.core.model.document.BaseUnit
 
 private[remod] object AMFResolver {
 
-  def resolve(bu:BaseUnit, env: BaseEnvironment):AmfResult = ???
+  def resolve(bu:BaseUnit, env: AMFConfiguration):AmfResult = ???
 
-  def resolve(bu:BaseUnit, pipeline: AMFResolutionPipeline, env: BaseEnvironment):AmfResult = ???
+  def resolve(bu:BaseUnit, pipeline: AMFResolutionPipeline, env: AMFConfiguration):AmfResult = ???
```

Comment:
- In which cases would I want to resolve with a pipeline that is not registered in the Configuration? 
- Why wouldn't I just run the pipeline myself?
- How do I choose the pipeline I want to run (or how is it chosen) ?

Reply:
This is going to be changed by @AgustinBettati . Today we talk about that.
Resolve method is going to recieve a BaseUnit, the configuration and a PipelineName

Reply:
Why not change it now?

Reply:
He is already working on that, and some analysis to check the arguments is being made.
On the other hand, the class is private for now, so no risk to have it merged.

Reply:
At the end, I changed as the class name also needed to be updated, so I took the opportunity.